src/minivhd_create.c

```diff
@@ -144,14 +144,20 @@ static int mvhd_gen_par_loc(MVHDSparseHeader* header,
      * This is the information needed to find the paths saved elsewhere
      * in the VHD image 
      */
+
+    /* Note about the plat_data_space field: The VHD spec says this field stores the number of sectors needed to store the locator path.
+     * However, Hyper-V and VPC store the number of bytes, not the number of sectors, and will refuse to open VHDs which have the 
+     * number of sectors in this field.
+     * See https://stackoverflow.com/questions/40760181/mistake-in-virtual-hard-disk-image-format-specification 
+     */
```

Comment:
Huh. I knew there was a discrepancy here, wasn't aware that some software would refuse to open a VHD if this wasn't set as the number of bytes.

Interesting, my belief is that the MS software is incorrect, not the spec itself. The idea in the spec was probably to ensure the space used was always sector aligned (since everything else about the VHD spec assumes sector alignment...)